### DISCUSSION TITLE
Please fix Gnome preview

### DISCUSSION THREAD

**Discussion by Jopp-gh** [linuxmint/discussions#604] [Ideas]
Gnome preview (ex Sushi I guess) is a very accessible, quick and useful and not to forget, awesome feature.

The current implementation has some problems however:
 - the window isn't centered, but placed with a shift to the bottom right. (Why ? Centered IS neutral, everything else IS "custom")
 - the window is too small. On desktop, the window geometry is just "crippled" , IMO.

### Solution
1. Show previews full height if the file height is bigger than the width
2. Show previews full width if the width is bigger than their height. 
3. Show previews centered on-screen. I don't get why a quick preview should be somewhere else on screen than in the "center of our interest"

**Comment by clefebvre** [comment]
This should be reported to GNOME directly.

**Comment by mtwebster** [comment]
This is nemo-preview

Open an issue there, more than likely there already is one.

https://github.com/linuxmint/nemo-extensions
